PictureApp.API.Services/UserService.cs

```diff
@@ -21,18 +23,37 @@ public UserService(IRepository<User> userRepo, IMapper mapper)
             _mapper = mapper ?? throw new ArgumentNullException(nameof(mapper));
         }
 
-        public UserForDetailedDto GetUser(int userId)
+        public async Task<UserForDetailedDto> GetUser(int userId)
         {
-            var user = _userRepository.Find(u => u.Id == userId).FirstOrDefault();
+            var users = await _userRepository.FindAsyncWithIncludedEntities(x => x.Id == userId, include => include.Photos);
+            var user = users.FirstOrDefault();
 
             if (user == null)
             {
                 throw new EntityNotFoundException($"user by id {userId} not found");
             }
 
-            return _mapper.Map<UserForDetailedDto>(user);
+            var userDto =_mapper.Map<UserForDetailedDto>(user);
+
+            return userDto;
         }
 
+        public async Task<UserForEditProfileDto> GetUserForEdit(int userId)
+        {
+            var users = await _userRepository.FindAsyncWithIncludedEntities(x => x.Id == userId, include => include.Photos);
+
+            var user = users.FirstOrDefault();
```

Comment:
We can move this section (lines: 45-50) to some common method as it is the same logic like above.